Makefile

```
@@ -169,7 +169,10 @@ codegen: ## Auto generate files based on AWS APIs response
 stable-release-pr: ## Generate PR for stable release
 	$(WITH_GOFLAGS) ./hack/release/stable-pr.sh
 
-release: ## Builds and publishes stable release if env var RELEASE_VERSION is set, or a snapshot release otherwise
+snapshot:
```

```suggestion
snapshot: ## Builds and publishes a snapshot release
```

Comment for what this command does